### ISSUE TITLE
Clarification on "language versions" within definition of "set of web pages"

### ISSUE THREAD

**Issue by joshueoconnor** [w3c/wcag#310]
Logged from  https://github.com/w3c/wcag21/issues/244 by @srodriguezvazquez 

Under the definition of "set of web pages", it reads:

  --> Different language versions would be considered different sets of Web pages.
I would like to suggest that a clarification on what is understood by "language version" (from a technical perspective) is included in WCAG 2.1, either here (under set of web pages) or as a new entry in the glossary. This would facilitate the decision-making process in terms of accessibility compliance in the case of multilingual websites. The rationale for this suggestion is that different localisation strategies may have been adopted to create parallel web content in different languages. When referring to localisation, I am following W3C's definition: https://www.w3.org/International/questions/qa-i18n

In WCAG-EM 1.0, language versions are considered as individual websites in terms of accessibility compliance:
Website in Multiple Versions
Some websites are available in multiple versions that are independent of one another in use, that is, using one version does not require or depend on using another version of the website. For example, a website may have a mobile version and there may be versions of a website in different languages that meet this characteristic. Usually each such website version has a different set of URIs. Such website versions can be considered as individual websites for evaluation.

This would be clearly the case in the following examples, as these language versions are independent of one another in use:

http://www.banquemondiale.org/ (FR) and http://www.worldbank.org/ (EN)
http://en.comune.fi.it/ (EN) and http://www.comune.fi.it (IT)
Would it be also the case when the different language versions share the same domain name? For instance:

Within www.canada.ca, we have: https://www.canada.ca/en.html (EN) and https://www.canada.ca/fr.html (FR)
www.ikea.com, which contains a global gateway to the different locales (www.ikea.com/es; www.ikea.com/ro, etc.)

**Comment by alastc** [comment]
Proposed response:
<hr>
It is possible to create different language versions at different URLs in several different ways, these are not particularly relevant to the conformance claim.

For example, a set of websites in different languages could:
- Use the same content management system (CMS) with the same domain name and use different "sub-folders" for different languages, **or**
- the same CMS could use  completely different domain names for each language version, **or** 
- the websites could use different systems (CMSs) under the same domain name with a CMS per sub-folder, **or**
- the websites could use different systems (CMSs) use  completely different domain names for each language version.

Each is a different set of web pages, and it is difficult to see how the wording could be made more specific given the complex systems that can be in place. 

As well as the technical differences due to each site being produced differenty, there are a lot of success criteria that can pass/fail with different wording, such as alt text, captions, sensory characteristics, text-spacing, page titled & link purpose.

So in answer to the last part of the comment: Yes, the sub-folder approach is considered a separate set of web pages.

**Comment by alastc** [comment]
The proposed response above was [accepted by the AG working group](https://www.w3.org/2002/09/wbs/35422/issues-2019-07/results#xq4) as the official response.
